From eaad8dfc2291f43b494dd2bb892b71466eeebb61 Mon Sep 17 00:00:00 2001
From: Stijn Tintel <stijn@linux-ipv6.be>
Date: Mon, 20 Jun 2022 19:25:22 +0300
Subject: [PATCH] hostapd: enable MBO if 802.11ax is enabled

Multi Band Operation is required for 802.11ax certification, so let's
enable it if 802.11ax support is enabled.

Signed-off-by: Stijn Tintel <stijn@linux-ipv6.be>
Acked-by: David Bauer <mail@david-bauer.net>
---
 package/network/services/hostapd/Config.in | 1 +
 1 file changed, 1 insertion(+)

diff --git a/package/network/services/hostapd/Config.in b/package/network/services/hostapd/Config.in
index e22bbff83a..4cfcaafade 100644
--- a/package/network/services/hostapd/Config.in
+++ b/package/network/services/hostapd/Config.in
@@ -78,6 +78,7 @@ config DRIVER_11AC_SUPPORT
 config DRIVER_11AX_SUPPORT
 	bool
 	default n
+	select WPA_MBO_SUPPORT
 
 config WPA_ENABLE_WEP
 	bool "Enable support for unsecure and obsolete WEP"
-- 
2.35.1

